
Bad key "text.kerning_factor" on line 4 in
/home/pmallya/anaconda3/envs/det2/lib/python3.6/site-packages/matplotlib/mpl-data/stylelib/_classic_test_patch.mplstyle.
You probably need to get an updated matplotlibrc file from
https://github.com/matplotlib/matplotlib/blob/v3.1.3/matplotlibrc.template
or from the matplotlib source distribution
Traceback (most recent call last):
  File "visualize_data_kitti.py", line 105, in <module>
    with open('./downlaods/val.txt') as val_file:
FileNotFoundError: [Errno 2] No such file or directory: './downlaods/val.txt'
